cmd/test-utils_test.go

```diff
@@ -366,6 +367,20 @@ func StartTestStorageRPCServer(t TestErrHandler, instanceType string, diskN int)
 	testRPCServer.AccessKey = credentials.AccessKey
 	testRPCServer.SecretKey = credentials.SecretKey
 
+	// Create dummy format json file.
+	emptyConfig := []byte("{}")
```

Comment:
It is since it allows to validate. The problem right is that no one will remember why this is done so in the first place.